R/derive_extreme_event.R

```diff
@@ -96,7 +96,10 @@
 #'       the number of the event.
 #'       1. Only the variables specified for the `keep_source_vars` field of the
 #'       event, and the by variables (`by_vars`) and the variables created by
-#'       `set_values_to` are kept.
+#'       `set_values_to` are kept. If `keep_source_vars = NULL` is used for an event
+#'       in `derive_extreme_event()` the value of the `keep_source_vars` argument of
+#'       `derive_extreme_event()` is used. In this case the `keep_source_vars` field of
+#'       the event will take precedence over the value of the `keep_source_vars` argument.
```

Comment:
Sorry, I think my QC comment was not clear.

I meant that we should add a note regarding precedence in the description of the `keep_source_vars` argument (line 72-74).
I think we do not need "In this case the `keep_source_vars` field of the event will take precedence over the value of the `keep_source_vars` argument." here.